### PR TITLE
NISRA routing change (HH/HI)

### PR DESCRIPTION
**Pull request by berroar**
### What is the context of this PR?
The routing rule has been changed for the `hours-worked` question (NISRA HH/HI). 

If `in-education` is `yes` then we should route to `study-location-type` rather than `work-location-type`.

### How to review
Check that the new routing rules match the overflow diagram attached to the trello card.

### Checklist

* [x] Jsonnet files conform to the latest [style guide](/ONSdigital/eq-questionnaire-schemas/blob/master/style_guide.md)

### Schemas Artifacts
Schemas artifacts are available at:
```bash
https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/nisra-routing-change/schemas/en/census_individual_gb_nir.json
```

### CODE REVIEW

**Comment by berroar** [comment]
[[A] NISRA routing changes (S)](https://trello.com/c/RrytXFxs/3831-a-nisra-routing-changes-s)

**Comment by MebinAbraham** [comment]
[NIR Individual Quick Launch](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-NIR&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/nisra-routing-change/schemas/en/census_individual_gb_nir.json)

File: source/jsonnet/northern-ireland/individual/blocks/employment/hours_worked.jsonnet [hunk 1]
```diff
@@ -76,4 +76,21 @@ local pastProxyTitle = {
...
+          value: 'Yes',
+        }],
+      },
+    },
+    {
+      goto: {
+        block: 'work-location-type',
+      },
+    },
+  ],
```
Comment (MebinAbraham) [inline source/jsonnet/northern-ireland/individual/blocks/employment/hours_worked.jsonnet:95]: Did a journey where I answered `Studying` and past job, but from hours worked I went to `work-location-type` but card says it should go to `study-location-type` if `in-education` is `yes` or `employment-type` is `studying`.

Comment (petechd) [reply source/jsonnet/northern-ireland/individual/blocks/employment/hours_worked.jsonnet:95]: I tried the same journey and it worked in my local environment?!

Comment (berroar) [reply source/jsonnet/northern-ireland/individual/blocks/employment/hours_worked.jsonnet]: Have fixed this, added an extra clause for employment-type

**Pull request merged**